src/main/perl/FileEditor.pm

```diff
@@ -58,13 +58,38 @@ sub _is_valid_source
     return -f $fn;
 }
 
+sub _is_reference_newer
+{
+    my ($self) = @_;
+    my $is_newer = 0;   # Assume false
+    if ( $self->_is_valid_source(*$self->{options}->{source}) ) {
+        if ( !$self->_is_valid_source(*$self->{filename}) || 
+             ((stat(*$self->{options}->{source}))[9] > (stat(*$self->{filename}))[9]) ) {
```

Comment:
10th enty is the mod time and should not be affected by the creation time in the RPM.